code/__HELPERS/unsorted.dm

```diff
@@ -1416,3 +1416,7 @@ GLOBAL_PROTECT(valid_HTTPSGet)
 		mob_occupant = brain.brainmob
 
 	return mob_occupant
+
+/proc/msglog_admins(text)
```

Comment:
I know it's not bad but this just seems lazy to me =P
At least put it in the logging helper file.